eng/Version.Details.xml

```diff
@@ -273,23 +273,23 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>59ca590949ade88c712e4ca8c6835acd0e9cbf46</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0-preview.4.20205.13" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0-preview.4.20205.13" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Actually should these be changed to Razor instead?

Reply:
If we have a direct subscription on dotnet/runtime, no. If it's still runtime -> tooling -> AspNetCore, then yes

Reply:
We should definitely **not** depend directly on dotnet/runtime. We hit problems almost every time the repo gets a newer runtime than what tooling built against.

Reply:
Right, I brain-farted and forgot we haven't done Tooling's consolidation yet. Yes, everything should be CPD'd thru AspNetCore-Tooling

Reply:
Okay, that's what I thought but I saw a bunch of CPDs in this file are set to NETCore.App for some reason and suspected we might have had a reason for it.

Reply:
Reason was history and this PR reflects the need to fix it up. Note as well that extensions, aspnetcore-tooling, ef6, efcore and extensions (anything that depends on dotnet/runtime packages) shouldn't need CPD at all in their 'master' branches -- assuming we're done taking anything _through_ extensions.